Blish HUD/GameServices/Input/Keyboard/TypedInputUtil.cs

```diff
@@ -14,142 +9,86 @@ internal static class TypedInputUtil {
         // because of this behavior, "^" is called dead key)
 
         [DllImport("user32.dll")]
-        private static extern int ToUnicodeEx(uint wVirtKey, uint wScanCode, byte[] lpKeyState, [Out, MarshalAs(UnmanagedType.LPWStr)] System.Text.StringBuilder pwszBuff, int cchBuff, uint wFlags, IntPtr dwhkl);
+        private static extern int ToUnicode(uint vkCode, uint scanCode, byte[] keyState, [Out, MarshalAs(UnmanagedType.LPWStr, SizeConst = 64)] System.Text.StringBuilder receivingBuffer, int bufferSize, uint flags);
 
         [DllImport("user32.dll")]
-        private static extern bool GetKeyboardState(byte[] lpKeyState);
+        private static extern uint MapVirtualKey(uint vkCode, uint mapType);
 
         [DllImport("user32.dll")]
-        private static extern uint MapVirtualKeyEx(uint uCode, uint uMapType, IntPtr dwhkl);
-
-        [DllImport("user32.dll", CharSet = CharSet.Auto, ExactSpelling = true)]
-        private static extern IntPtr GetKeyboardLayout(uint dwLayout);
-
-        [DllImport("User32.dll")]
-        private static extern IntPtr GetForegroundWindow();
-
-        [DllImport("User32.dll")]
-        private static extern uint GetWindowThreadProcessId(IntPtr hWnd, out uint lpdwProcessId);
+        private static extern bool GetKeyboardState(byte[] keyboardState);
 
         [DllImport("user32.dll")]
-        private static extern bool AttachThreadInput(uint idAttach, uint idAttachTo, bool fAttach);
-
-        [DllImport("kernel32.dll")]
-        private static extern uint GetCurrentThreadId();
+        private static extern byte GetKeyState(int keyState);
 
-        private static uint   _lastVkCode   = 0;
-        private static uint   _lastScanCode = 0;
-        private static byte[] _lastKeyState = new byte[255];
-        private static bool   _lastIsDead   = false;
+        private static uint _lastVirtKeyCode;
+        private static uint _lastScanCode;
+        private static byte[] _lastKeyState = new byte[256];
+        private static bool _lastIsDead;
+        private static byte VK_SHIFT = 0x10;
+        private static byte VK_MENU = 0x12;
```

Comment:
These are walking and talking like consts.  Let's make 'em consts and place them near the top of the file (either before or after the DLLImports is fine).